ee/vellum_ee/workflows/display/types.py

```diff
@@ -54,6 +54,7 @@ class WorkflowDisplayContext:
     _errors: List[Exception] = field(default_factory=list)
     _invalid_nodes: List[Type[BaseNode]] = field(default_factory=list)
     _dry_run: bool = False
+    generated_files: Dict[str, str] = field(default_factory=dict)
```

Comment:
[nit] instead of threading through the original files, could we use a similar approach that `virtual_open` is using here? https://github.com/vellum-ai/vellum-python-sdks/pull/2416/files#diff-57899bd571a4ab4e7aab18a151b0b0390644e4607b3fc95f4b115299704c52f8R68-R74